### DISCUSSION TITLE
How can I fix "Present(No Resource)" issue in renderDOC

### DISCUSSION THREAD

**Discussion by techpond31** [eliemichel/MapsModelsImporter#293] [Q&A]
Now I am trying to capture google map data to convert it 3D model but it's not working.
I have captured google map using D3D11 API but after capturing I cant get resource of capture.
How can I fix it?

Attached is my renderDOC window with issue.
![image](https://github.com/eliemichel/MapsModelsImporter/assets/86774429/d6cd6162-4981-4268-8a8a-aca46adb708f)

@eliemichel 
I have followed your article and thank you for your posting.
But now I am getting issue like the above so please help me how can I pass this issue.

Looking forward to hearing from you.
Best regards.

**Comment by eliemichel** [comment]
The D3D11 API was presenting during the capture but is not during replay? Do you replay on the same machine? Make sure RenderDoc and Chrome run on the same GPU if you have a laptop with integrated+dedicated chips.

**Reply by techpond31** [reply]
@eliemichel 
Thanks for your reply.
I will check it again and give you answer.

**Comment by Japol1379** [comment]
Hello everyone,
I’m trying to do the same thing. 3d print a city. But it won’t let let me due to renderdoc saying: “present (no resource)”
My D3D11 is active, I can save the file, but when I upload into blender I get nothing. I just get an empty folder that appears in my downloads
Can anybody help with this? Again, it’s the same issue I’m having with the original poster.
![image](https://github.com/eliemichel/MapsModelsImporter/assets/162852805/2d66db1a-f560-484f-9173-9e05c88ec796)
